Added semicolon to the end of the PackageTargetFallback

seancpeters opened the pull request.

seancpeters committed:
Added semicolon to the end of the PackageTargetFallback

dnfclas commented:
Hi **@seancpeters**, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by .NET Foundation and real humans are currently evaluating your PR.</p>

TTYL, DNFBOT;

seancpeters committed:
Commented out all instances of AddUserSecrets()
This is a temporary change, until we get a new package to handle this
functionality.

seancpeters commented:
Also commented out all instances of AddUserSecrets() - this is a temporary change.
